http/stats.md

```diff
@@ -0,0 +1,54 @@
+# HTTP Stats
+
+All views and measures have the prefix: "opencensus.io/http". 
+
+All times (latencies) are measured in float64 fractions of microseconds.
+
+There is no special support for multi-part HTTP requests and responses. These are just treated as one request.
+
+## Client
+
+All views, measures and tags should have the prefix: "opencensus.io/http/client". For example: "opencensus.io/http/client/started".
+
+### Measures
+
+Client stats are recorded for each individual HTTP request, including for each redirect (followed or not). 
+
+| Measure suffix     | Default View Aggregation | Description                                                                                                                                                                                                                                                                                       |
+|--------------------|--------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| started            | sum                      | Number of all client requests started                                                                                                                                                                                                                                                             |
```

Comment:
I would not add this for the moment, or maybe just mark it as optional.

Reply:
The way I see it, everything is optional. I meant this as more like if the language provides this metric then this is what it's name should be. Curious why you think this is not an important metric?

Reply:
I did not want to say that it is not important, in general we get this information from the tracing side by looking at the number of active HTTP spans.

Happy to keep it as optional (as you mentioned all the metrics are optional).

FYI: also in general for the number of completed requests we get that from the latency distribution by querying the total count you can mentioned that in this doc.

Reply:
Yes, this is how the view "ended" is defined below.